dockerfiles/run/guice/destination/conf/elasticsearch.properties

```diff
@@ -26,3 +26,8 @@ elasticsearch.nb.shards=1
 elasticsearch.nb.replica=0
 elasticsearch.retryConnection.maxRetries=7
 elasticsearch.retryConnection.minDelay=3000
+
+# This properties defines the maximal length of search results. Higher values
+# means more results but costs more performances.
+# Default value is 100
+elasticsearch.search.size=100
```

Comment:
Ok, let speak a bit of ElasticSearch
See for instance https://www.elastic.co/guide/en/elasticsearch/reference/current/search-uri-request.html
You might find the size parament astounding : "The number of hits to return. Defaults to 10."
But... Why do ElasticSearch do that ?
Well, because of scalability concerns. Every shards returns its hits to the node performing the request. This node have to sort this list by relevance. As people are mostly concerned in general by search top results, a good optimisation is to limit the number of results returned (say 10) so that each shard can return its top 10 results, that will get ordere and trouncated on the node performing the query.
You have to give a size to your searches.
One workaround is using "Scrolls". Scrolls allow you to take back were you stopped.

For instance, with a query size of 10 : 
 
 -  step 1 get 10 first result of each shards and merge them : take 0->10
 - step 2 get 20 first results of each shards and merge them : take 10->20

 and so on.

By the way, in the first lines of the scroll API you have something that sounds not good : Scrolling is not intended for real time user requests, but rather for processing large amounts of data, e.g. in order to reindex the contents of one index into a new index with a different configuration.

We didn't implemented search using scroll API by lack of s/time/will/ . I remember having said it.

Well, this is for limiting response size of search in IMAP. To sum up, rapid implementation + technical limitation of ElasticSearch

Then for exposing to the configuration : 
 - I consider the more configuration you expose to the ADMIN, the better. Here he can trade accuracy vs performances.

I understand we disagree on the need to expose as many stuff as possible to configuration (of course with decent messages and a default value to back it up).

So I removed this change set.

But the size limitation will remain, with a constant value setted in some Java file.